core/metadata.js

```diff
@@ -360,10 +360,14 @@ const makeComparator = (name, interestingFields) => {
       })
     })
   }
+  const isRealDiff = (difference) => {
```

Comment:
I'm not pleased by the name `isRealDiff`. As I understand, this function returns `false` when we have a "real" diff. Maybe `canBeIgnoredDiff`?

Reply:
:+1: for new name and good to merge afterward